### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by dockrieger86**
Registry request for ANVIL Pool

### CODE REVIEW

**Comment by dockrieger86** [comment]
hi, just wanted to see if my registration request has gone through or if I need to change something.
THanks

File: registry.json [hunk 1]
```diff
@@ -2012,5 +2012,20 @@
...
+    "poolId":"9084ebeb011e610e695068a74fa883dac5eda55faca77107b5f40064",
+    "social":{
+      "facebook":"",
+      "twitter":"https://twitter.com/AnvilPool",
+      "website":"https://anvilpool.carrd.co/",
+      "reddit":"",
+      "instagram":"",
+      "youtube":""
+    },
+    "telegram":"",
```
Comment (leo42) [inline registry.json:2027]: We need a active personal telegram handle (no groups or pages)

**Review by leo42 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
@dockrieger86 please can you update this PR or we will close? Thanks

**Comment by Bez625** [comment]
Closing as we believe this pool was renamed any way

**Pull request closed without merging**